src/main/java/seedu/address/model/company/Phone.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.model.company;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's phone number in the address book. TODO: Javadocs (Shawn)
+ * Guarantees: immutable; is valid as declared in {@link #isValidPhone(String)}
+ */
+public class Phone {
+
+
```

Comment:
Perhaps an accidental double lines here?

Reply:
My bad, thanks for catching this!